AutomaticInterface/AutomaticInterface/Builder.cs

```diff
@@ -9,13 +9,13 @@ namespace AutomaticInterface;
 
 public static class Builder
 {
-    private const string InheritDoc = "/// <inheritdoc />"; // we use inherit doc because that should be able to fetch documentation from base classes.
+    private static string InheritDoc(string source) => $"/// <inheritdoc cref=\"{source.Replace('<', '{').Replace('>', '}')}\" />"; // we use inherit doc because that should be able to fetch documentation from base classes.
```

Comment:
Might it be tidier if this method took a symbol instead of a string? I feel it would give better type safety and single point of control for conversion of methods/whatever to crefs.

I'm not the owner of this repo or anything, so it's just an idea :)

Reply:
Yep, valid.. I see there is a common `ISymbol` interface that I can pass from all three calls

Reply:
This is resolved